### ISSUE TITLE
Miss-indentation of keyword arguments

### ISSUE THREAD

**Issue by tkf** [domluna/JuliaFormatter.jl#190]
```julia
(v1.x) pkg> st JuliaFormatter
    Status `~/.julia/environments/v1.x/Project.toml`
  [00ebfdb7] CSTParser v2.1.0
  [98e50ef6] JuliaFormatter v0.3.3
  [0796e94c] Tokenize v0.5.7

julia> using JuliaFormatter

julia> """
       begin
           fffffffffffffff(ggggggggggggg(@inbounds aaaaa[i]), bbbbbbbbbbbb; c = dddddddddddd, kw...)
       end
       """ |> format_text |> print
begin
    fffffffffffffff(ggggggggggggg(@inbounds aaaaa[i]), bbbbbbbbbbbb; c = dddddddddddd,
    kw...)
end
```

I think the indentation of `kw...)` is using something not intended.

**Comment by domluna** [comment]
This looks to be related to #174

**Comment by domluna** [comment]
This looks to be have been fixed as collateral of #198. 

Not that I'm complaining :smiley:

**Comment by domluna** [comment]
looks fixed on 0.4
